### DISCUSSION TITLE
Adding local sounds

### DISCUSSION THREAD

**Discussion by KristalAndStarlight** [jenius-apps/ambie#399] [Q&A]
So I installed this app because it was open source, but I cannot seem to be find a way to add custom sounds, I navigated my way to `C:\Program Files\WindowsApps\43891JeniusApps.Ambie_4.0.12.0_x64__jaj7tphbgjeh8\Assets\Sounds`
but don't seem to be able to move files in or out even with administrator access.
Anyone knows what to do?

**Comment by dpaulino** [comment]
What sounds are you wanting to add? Maybe I can add it to the catalogue so everyone can benefit?

**Reply by KristalAndStarlight** [reply]
https://github.com/opera-gaming/gxmods/blob/main/mods/Rain_Ambience/music/rain.mp3
but iirc by default the license is ARR on github, I am gonna try music on console if there is no way to upload your own sounds
nevermind music on console is unix only
